pr_toolbox/quantum/results/frequencies.py

```diff
@@ -62,3 +63,16 @@ def bitmask_counts(counts: Counts, bitmask: int) -> Counts:
         New counts with readout bits masked according to input.
     """
     return map_counts(counts, lambda readout: readout & bitmask)
+
+
+def counts_to_quasi_dists(counts: Counts) -> QuasiDistribution:
+    """Infers a :class:`~qiskit.result.QuasiDistribution` from :class:`~qiskit.result.Counts`.
+    Args:
+        counts: the counts to convert.
+    Returns:
+        New QuasiDistribution inferred from counts.
+    """
+    shots = counts.shots()
```

Comment:
I forgot one thing here: we need to validate that the input is indeed counts and raise a `TypeError` if it is not 😓 the current error that we would get would be cryptic I think...